packages/@sanity/portable-text-editor/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "../../../tsconfig",
+  "include": ["src/**/*"],
+  "compilerOptions": {
+    "composite": true,
+    "rootDir": "./src",
+    "outDir": "./dist/dts",
+    // todo: we need to disable strict in a transition phase and should be removed eventually
```

Comment:
is it intentional to keep these commented out? maybe remove if possible?

Reply:
Cleaned up.